encrypted-media/scripts/setmediakeys-multiple-times-with-different-mediakeys.js

```diff
@@ -53,34 +53,35 @@ function runTest(config, qualifier) {
             // Set mediaKeys2 on video element (switching MediaKeys) need not
             // fail after src attribute is set.
             return _video.setMediaKeys(_mediaKeys2);
-        })).then(test.step_func(function() {
+        }).then(test.step_func(function() {
             // Switching setMediaKeys after setting src attribute on video element
             // is not required to fail.
-            assert_equals(_video2.mediaKeys, _mediaKeys2);
             fail = false;
+            assert_equals(_video.mediaKeys, _mediaKeys2);
             return Promise.resolve();
-        }, test.step_func(function(error) {
+        }), test.step_func(function(error) {
             fail = true;
             assert_equals(_video.mediaKeys, _mediaKeys1);
             assert_in_array(error.name, ['InvalidStateError','NotSupportedError']);
```

Comment:
Isn't there only one correct exception? I'm fine with having this for the purposes of making progress, but we should have an open issue and reference it here.